DANS-KNAW/easy-stage-dataset
# Fix easy 1195

fixes EASY-1195 bug: file-location wordt genegeerd in easy-stage-file-item

#### When applied it will
* use the file-location when specified
* do proper verification of the input parameters and require a format when a file is supplied

#### Where should the reviewer @DANS-KNAW/easy start?
easy-stage-dataset/src/main/scala/nl/knaw/dans/easy/stage/fileitem/FileItemConf.scala

#### How should this be manually tested?
* build an deploy easy-stage-dataset 
* then run it with the supplied files in the test resources 
`--csv-file file_properties-test.csv outdir`
* The 'outdir' should be created and contain a valid SDO when the command is run
